### PR TITLE
Migrating to a cluster capacity script that uses the kubernetes clien…

### PR DESCRIPTION
**Pull request by etsauer**
…t library

This script replaces the current capacity script with one that uses the python k8s client. This makes the script much simpler, and more flexible, as it can be run from any logged in client (as opposed to only from a master).

### CODE REVIEW

File: cluster-management/README.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
 
 This directory contains scripts specific to cluster managment intended to be run after an OCP installation for operationalization purposes.
 
-### Contents:
+## Cluster Capacity Report (cluster_capacity.py)
 
-1. Garbage collection
-- Sets up spotify docker-gc on hosts. Should be run a standard 'bring your own' OpenShift-ansible inventory file. The playbook will set up garbage collection on all host with the [node] designation. 
-- View the documentation for spotify docker-gc here: [LINK](https://github.com/spotify/docker-gc)
+The `cluster_capacity.py` script generates a report showing how full a kubernetes cluster is. It can be run from anywhere there is an active `~/.kube/config` context and requires at least `cluster-reader` or equivalent permissions.
```
Comment (sabre1041) [inline cluster-management/README.md]: Can you mention that this script is in the `capacity` folder. or add a `cd capacity` instruction in the `README.md`

File: cluster-management/README.md [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+
+The script can be run with no arguments, but must be run with Python 3.
+
+```
+python3 cluster_capacity.py
```
Comment (sabre1041) [inline cluster-management/README.md]: Can you make the script executable?

Comment (etsauer) [reply cluster-management/README.md]: Done

File: cluster-management/README.md (hunk 1 above)
Comment (etsauer) [reply cluster-management/README.md]: Done

**Comment by huddlesj** [comment]
Tested on three different clusters. All were Openshift 3.10.

**Comment by sabre1041** [comment]
LGTM

**Pull request merged**